src/platform/packages/shared/kbn-visualization-utils/src/get_timezone.test.ts

```diff
@@ -28,7 +28,14 @@ describe('getTimeZone', () => {
     expect(
       getTimeZone({
         get: () => 'Browser',
-        isDefault: () => true,
+      } as unknown as IUiSettingsClient)
+    ).toEqual('America/New_York');
+  });
+
+  it('returns local time zone when uiSettings returns Browser as non default', () => {
+    expect(
+      getTimeZone({
+        get: () => 'Browser',
```

Comment:
```suggestion
```

Two similar tests, no?

Reply:
You are right, removed the test here [1022968](https://github.com/elastic/kibana/pull/220658/commits/10229680cec2a3f027a4420af826d4fd7d0561fa)